src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -2725,6 +2724,9 @@ def __init__(self, io_backend: str = "disk", num_threads: int = 1, **kwargs) ->
 
     def _get_video_reader(self, filename: str) -> decord.VideoReader:
         if self.file_client is None:
+            # TODO(wonjulee): Remove mmengine imports
+            from mmengine.fileio import FileClient
```

Comment:
@wonjuleee I think this is missing from the recent action task mm decoupling. I guess if this is used, action still has mm dependency, can you confirm?

Reply:
For now, let's merge this PR.